### ISSUE TITLE
Liquid Exception: tried to create a Proc object without a block

### ISSUE THREAD

**Issue by grantwforsythe** [inukshuk/jekyll-scholar#327]
I am running into an issue when I try to build my Jekyll site locally. The error message is as follows:
```console
Liquid Exception: No such file or directory @ rb_sysopen - C:/Users/grant/Desktop/Repos/www/./_bibliography/references.bib in C:/Users/grant/Desktop/Repos/www/_posts/2021-03-26-post-title.md
jekyll 3.9.0 | Error:  No such file or directory @ rb_sysopen - C:/Users/grant/Desktop/Repos/www/./_bibliography/references.bib
C:/Ruby30-x64/lib/ruby/gems/3.0.0/gems/bibtex-ruby-4.4.7/lib/bibtex/bibliography.rb:150:in `new': tried to create Proc object without a block (ArgumentError)
```
Could this be the result of a version conflict with some plugins?

The source code can be found [here](https://github.com/grantwforsythe/www).

**Comment by inukshuk** [comment]
Yes, you need to update your locally installed Gems.

**Comment by grantwforsythe** [comment]
> 
> 
> Yes, you need to update your locally installed Gems.

I used `bundle update` to update the gems locally, but it's still throwing me the same error.

**Comment by inukshuk** [comment]
Can you confirm that you're on jekyll-scholar 6.8.1 and bibtex-ruby 6.0.0 or 5.1.7? (`bundle update` may not necessarily update to the latest versions, depending on what you have specified in the Gemfile)

**Comment by grantwforsythe** [comment]
It appears I was not. At the time, I was using `bibtex-ruby 4.47` and `jekyll-scholar 5.16.0`. 

I since have installed the correct versions with `gem install`. Do I have to specify the exact versions within the `Gemfile` and run `bundle update` as well?

**Comment by inukshuk** [comment]
My apologies, I guess we never released 5.1.7! Anyway, I pushed a new jekyll-scholar version that depends on bibtex-ruby 6.0 or later so you should be able to update now.

**Comment by grantwforsythe** [comment]
Yes, that has resolved the issue.

Thank you.
